cape_encrypt/cape_encrypt/cape_encrypt.py

```diff
@@ -0,0 +1,79 @@
+"""
+Cape Encrypt functionality for use within functions deployed to run in a Cape Enclave.
+
+This provides the ability to be able to encrypt or decrypt information solely within a
+Cape Enclave using the Cape Key associated with the function owner. Functions can then
+encrypt data generated during the execution of the function or have fine grained
+control over decrypting inputs.
+"""
+import json
+import socket
+
+
+def connect() -> socket.socket:
+    """Connect to unix socket on a Cape Enclave to request operations with the Cape
+    Key
+    """
+    sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    sock.connect("../rpc.sock")
+    return sock
+
+
+def encrypt(plaintext: str) -> str:
```

Comment:
what is the reason for writing this function to operate on strings instead of bytes? in PyCape the cape.encrypt functionality is a function mapping `bytes->bytes` ([here](https://github.com/capeprivacy/pycape/blob/f1d021bd524688fe32e4efdeb0c18ee3cb5c8965/pycape/cape.py#L128-L134)). i would assume we'd want to keep it that way, but maybe i'm missing something elsewhere

Reply:
Updated